task1/src/components/HelloWorldCreateElement.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const HelloWorldCreateElement = React.createElement(
+    'div',
+    {},
+    React.createElement('h1', {}, 'Hello World!')
+);
+
+export default HelloWorldCreateElement
```

Comment:
`HelloWorldCreateElement` is never used. Is it ok?)

Reply:
Hi! Thank you for your reply! Actually, i tried to use it in index.js file. But there is only one ReactDOM.render is available. Or should I wrap it to function?  ![image](https://user-images.githubusercontent.com/22503765/107372571-c0e75680-6b0f-11eb-8ce6-6a62f14c7adf.png)

Reply:
Thank you for answer, I think no need to change anything considering it is test app, everything is top.